app/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepository.scala

```diff
@@ -137,21 +133,6 @@ class CaseMongoRepository @Inject()(mongoDbProvider: MongoDbProvider, mapper: Se
     removeAll().map(_ => ())
   }
 
-  override def incrementDaysElapsed(increment: Double): Future[Int] = {
```

Comment:
@OC-Pomaroff was this just some leftover stuff where they were attempting to just add an extra day to all of the open cases each time?

Reply:
I think so! The code was not used anywhere except the tests, so to avoid any confusion I thought it best to remove the code.